bedrock/firefox/views.py

```diff
@@ -560,6 +560,7 @@ def download_thanks(request):
     experience = request.GET.get('xv', None)
     locale = l10n_utils.get_locale(request)
     variant = request.GET.get('v', None)
+    show_newsletter = locale in ['en-US', 'en-GB', 'en-CA', 'en-ZA', 'es-ES', 'pt-BR', 'fr', 'ru', 'id', 'de', 'pl']
```

Comment:
@ejregithub can you please double-check the list of locales here that should get shown the newsletter? We're including all English locales (assuming that's wanted). We could also show all the `es` locales too if wanted.